source/deployment/elasticsearch.rst

```diff
@@ -8,13 +8,12 @@ Elasticsearch (E20)
 .. toctree::
     :maxdepth: 2
 
-Deployment Guide
```

Comment:
Looks good, expect I think this heading doesn't necessarily need to be removed. 0/5

Reply:
Re-reading the document now and I think it should be re-added and moved to just below the first note.